python/TestHarness/StatusSystem.py

```diff
@@ -62,7 +63,7 @@ class StatusSystem(object):
     fail = status(status='FAIL', color='RED', code=0x80)
     diff = status(status='DIFF', color='YELLOW', code=0x81)
     deleted = status(status='DELETED', color='RED', code=0x83)
-    error  = status(status='ERROR', color='RED', code=0x80)
+    error  = status(status='ERROR', color='RED', code=0x84)
```

Comment:
0x80 was by design and not a typo. If we are trying to be more specialized, I have no problems with it. I am not using 'code' during logical status comparisons so I suppose this is alright.

I do notice you are attempting to add a new status. But I do not see a new status tuple?

Reply:
Ok, I was just making it reflect the part above where 'fail' has the 0x80 code and error has the code 0x84.  Figured two statuses using the same code would make using the code less useful for identifying the root cause or something like that.  If not, I would propose changing the above statement to read that the 'error' exit code is 0x80.

As for the status I was adding, I added a description and the exit code that I am using just as an informational reference.  As my code stands, it wouldn't be using the tuple, so I wasn't sure if I should add one that never got used.